src/assets/icons.ts

```diff
@@ -29,3 +29,4 @@ export { default as WeChat } from './wechat.svg?react';
 export { default as Youtube } from './youtube.svg?react';
 export { default as GreenCircle } from './icon-green-circle.svg?react';
 export { default as RedCircle } from './icon-red-circle.svg?react';
+export { default as MagificentSeven } from './magnificent-seven.svg?react';
```

Comment:
I wouldn't put this here please, instead do a direct import inside only the story, we don't want to publish this icon for downstream consumers